src/qibolab/qubits.py

```diff
@@ -3,19 +3,19 @@
 
 import numpy as np
 
-from qibolab.channels import Channel
+from qibolab.components import AcquireChannel, DcChannel, IqChannel
 from qibolab.couplers import Coupler
 from qibolab.native import SingleQubitNatives, TwoQubitNatives
 
 QubitId = Union[str, int]
 """Type for qubit names."""
 
-CHANNEL_NAMES = ("readout", "feedback", "drive", "flux", "twpa")
+CHANNEL_NAMES = ("measure", "acquisition", "drive", "flux")
```

Comment:
Why *measure* instead of *readout*? Wasn't it pretty standard?

About *acquisition* instead, I fully agree.

Reply:
readout is the process of reading out the qubit state, which includes acquisition as well. However, I myself am not that happy with the word "measure", if there are any better suggestions, please let me know. In experimental physics the process of sending the pulse to the readout line is sometimes called "probing", so we could name it "probe" as well, but I can guarantee a lot of questions from the lab for the following two months like "can you remind me what is probe again?" :D  

Another problem with current naming is that some of them are verbs (drive, measure), some of them are noun (flux, acquisition). It would be nice to find a common ground in this respect as well. I am open to suggestions.

Reply:
> readout is the process of reading out the qubit state, which includes acquisition as well.

I would agree with this. But I would say that `measure` is a perfect synonym in this case.
So, in lack of a better alternative, I'd keep the previous one. But I'd also like an alternative, and `probe` could be a good one (probe is the correct word even outside the quantum field, and even outside physics itself).

> Another problem with current naming is that some of them are verbs (drive, measure), some of them are noun (flux, acquisition). It would be nice to find a common ground in this respect as well. I am open to suggestions.

Though I'd also prefer consistency, I'd say it's not a priority.
Moreover, this is somehow consistent, because all of them are used as adjectives for channels (*drive channel*, *flux channel*, ...). So, what it matters grammatically is the role in which the word is used.

Reply:
Let's try to establish `probe`, it definitely makes sense.